data/human/transport missions.txt

```diff
@@ -3582,3 +3582,743 @@ mission "Hauler VI cargo prototype 3"
+				goto inorout
+			label wasok
+			`	"Not exactly a thrilling review, but you're entitled to your opinion," she says smiling.`
+				goto inorout
+			label political
+			`	"Oh. Well unfortunately for many, everything is political. It was political when the Republic tried to block the evacuation. And it was political when the Navy did it without permission. And the same goes for every single act of oppression and liberation since the dawn of humanity. Our oppressors are very political. Maybe you have the hope of being one some day; wealthy, authoritative. Or maybe you just don't care. Either way, you don't sound like a good fit for this project. But thanks for transporting the installation," she says with a smile. She gets her smoothie to-go and walks out.`
+				decline
+			label inorout
+			`	Mariel said you saw me take the photos of the woman and her baby," you are surprised by the bluntness of her admission. "I really like that moment; not many notice it. You're observant but are you kind? We need a ship for our next project but I can't tell you much until you accept. We need you to transport a team to one planet, quickly land on another, then pickup the team and return here with everyone safe and sound. Nothing to dicey. But it's a dangerous universe, so who knows. Oh, and we have to leave today. It will pay, of course. 200000 credits. So what do you say? Are you in or out?" Your orders are served and you eat in silence and consider her proposition while she drinks her smoothie.`
+			choice
+				`	"In."`
+				`	"Out."`
+					goto imout
+			`	"Great. My name is Alejandra, but call me Ale. Here's the plan: you're going to drop me and a friend off at Maker where we are going to film in 3D the conditions in a secret Syndicate sweatshop. Thing is these aren't just poorly paid factory workers, but slaves. As you can imagine we can't get in as regular workers so with a bit of hacking we got ourselves jobs on site as private security. We know the Syndicate is getting the slaves from the pirates on Covert, and we also know where they do the transaction. So while we are getting the footage we need in the factories you will be going with our associate Bunker to spy on the Syndicate's rendezvous with the pirates on Hippocrates."`
+			`    "Don't worry," she reassures you, sensing your concern about spying on pirates and the Syndicate. "Bunker is a pro and we have an array of long range stealth drones that do the spying, you won't be anywhere near them. But pirates do frequent the system so be quick."`
+			`	See waves to the server for the bill as she finishes the last gulp of her smoothie. "Let's go. Everyone will be meeting at the hanger shortly."`
+			`	You go back to your ships hanger and sure enough there are two people with backpacks waiting for you with a few cases. They introduce themselves as River and Bunker.`
+			choice
+				`	"I am Captain <first> <last>."`
+				`	"I'm <first>."`
+					goto loadup
+			`	They all grimace at your formality.`
```

Comment:
This bit doesn't really add any substance tp the story. You might want to consider taking it out.